### PR TITLE
Updating GRPC Server Auth test docs to use MetaData.Key

### PR DESCRIPTION
**Pull request by tmulle**
This is my PR for https://github.com/quarkusio/quarkus/issues/43811

### CODE REVIEW

File: docs/src/main/asciidoc/grpc-service-implementation.adoc [hunk 1]
```diff
@@ -521,29 +521,47 @@ quarkus.http.auth.basic=true <1>
 ----
 package org.acme.grpc.auth;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
 
+import com.rajant.connector.proto.Greeter;
```
Comment (cescoffier) [inline docs/src/main/asciidoc/grpc-service-implementation.adoc:527]: Should we use a more neutral package name? Like org.acme?

**Review by cescoffier — APPROVED** [review]
Thanks!

I just one small comment about the package name used for the stubs.

File: docs/src/main/asciidoc/grpc-service-implementation.adoc (hunk 1 above)
Comment (tmulle) [reply docs/src/main/asciidoc/grpc-service-implementation.adoc:527]: Oops yes sorry that should be org.acme
I’ll fix it in a few mins

Comment (tmulle) [reply docs/src/main/asciidoc/grpc-service-implementation.adoc]: Fixed

**Review by cescoffier — APPROVED** [review]

**Comment by cescoffier** [comment]
Can you squash your commits into a single one?

**Comment by tmulle** [comment]
will open a new PR

**Pull request closed without merging**